modules/system/defaults/NSGlobalDomain.nix

```
@@ -7,6 +7,21 @@ let
   inherit (config.lib.defaults.types) floatWithDeprecationError;
 in {
   options = {
+    system.defaults.NSGlobalDomain.AppleActionOnDoubleClick = mkOption {
+      type = types.nullOr (
+        types.enum [
+          "Minimize"
+          "Zoom"
```

For me (macOS 15.5), even though System Settings shows "Zoom", the actual value used by defaults is `Maximize`
```suggestion
          "Maximize"
```

---

I think it would be good to use `apply` here to map from `Zoom` to `Maximise`